app/assets/javascripts/application.js

```diff
@@ -13,6 +13,8 @@
 //= require vendor
 //= require about
 //= require proxy_status
+//= require proxy_status/puppet
+//= require proxy_status/logs
```

Comment:
How do I do this properly? We already load bunch of stuff here and I believe not loading it breaks the TurboLinks thing... Teach me please.